### PR TITLE
Reworded/fixed grammatical error

### LINKED ISSUE #708: Grammatical error in the commented part
## Description
There is a grammatical error in the commented part of [exceptions.py](https://github.com/python-discord/sir-lancebot/blob/main/bot/utils/exceptions.py) 

## Possible Solutions
To reword/fix the comment.

## Would you like to implement a fix?
- [x] I'd like to implement the bug fix
- [ ] Anyone can implement the bug fix

### PR DESCRIPTION
**Pull request by chincholikarsalil**
### Relevant Issues
Closes #708 

## Description
Fixed the grammatical error in the commented part of [exceptions.py](https://github.com/python-discord/sir-lancebot/blob/main/bot/utils/exceptions.py)

## Did you:
- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

File: bot/utils/exceptions.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 class UserNotPlayingError(Exception):
-    """Will raised when user try to use game commands when not playing."""
+    """Will be raised when the users try to use game commands when they are not playing."""
```
Comment (jb3) [inline bot/utils/exceptions.py:2]: I think this wording is better.
```suggestion
    """Raised when users try to use game commands when they are not playing."""
```

**Review by jb3 — CHANGES_REQUESTED** [review]

**Review by ChrisLovering — APPROVED** [review]
New version reads good to me. Nice change :)

**Review by dawnofmidnight — APPROVED** [review]
I like how clear this sounds!

**Comment by chincholikarsalil** [comment]
@jb3 Please check

> New version reads good to me. Nice change :)

@ChrisLovering Thanks a lot :)

**Review by jb3 — APPROVED** [review]

**Pull request merged**